TopMovies/Core/State/CategoriesPaginationState.swift

```diff
@@ -0,0 +1,84 @@
+//
+//  CategoriesPaginationState.swift
+//  TopMovies
+//
+//  Created by Macbook Pro  on 19.02.2021.
+//
+
+import ReSwift
+
+struct CategoriesPaginationState: StateType {
+  typealias CategoryState = PaginatedOperationState<Movie.ID, Int>
+  let paginated: [MovieCategory.ID: CategoryState]
+}
+
+extension CategoriesPaginationState.CategoryState: Defaultable {
+  typealias Value = CategoriesPaginationState.CategoryState
+  static var defaultValue: Value {
```

Comment:
You can write 
```swift
static var defaultValue: CategoryState {
```
since Defaulteble will inherit a type for `Value` from the implemented variable.